pkg/base/linuxutils/lsblk/lsblk.go

```diff
@@ -178,17 +178,11 @@ func (l *LSBLK) SearchDrivePath(drive *api.Drive) (string, error) {
 	vid := drive.VID
 	pid := drive.PID
 	for _, l := range lsblkOut {
-		if strings.EqualFold(l.Serial, sn) && strings.EqualFold(l.Vendor, vid) &&
-			strings.EqualFold(l.Model, pid) {
-			device = l.Name
-			break
+		if strings.EqualFold(l.Serial, sn) {
+			return l.Name, nil
 		}
 	}
 
-	if device == "" {
-		errMsg := fmt.Errorf("unable to find drive path by SN %s, VID %s, PID %s", sn, vid, pid)
-		return "", errMsg
-	}
-
-	return device, nil
+	errMsg := fmt.Errorf("unable to find drive path by SN %s, VID %s, PID %s", sn, vid, pid)
```

Comment:
The message should be rephrased now, it is misleading, now we're searching only by serial number.
Please adjust also function comment accordingly.